x-pack/plugins/observability_solution/logs_explorer/common/data_views/models/data_view_descriptor.test.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataViewDescriptor } from './data_view_descriptor';
+
+describe('DataViewDescriptor', () => {
+  it('should correctly assert whether a data view has "logs" type', () => {
```

Comment:
**note**: this is a unitary test against different data view titles to verify whether it correctly identifies a data view as of `logs` type. If some entries which don't cover specific cases should be here, please add any entry here and I'll add it, thanks in advance!